test/test_utilities.cpp

```diff
@@ -0,0 +1,213 @@
+#include <iostream>
+#include <chrono>
+#include <future>
+#include <thread>
+#include <condition_variable>
+#include "gtest/gtest.h"
+#define SRT_TEST_CIRCULAR_BUFFER
+#include "api.h"
```

Comment:
Useless `#define SRT_TEST_CIRCULAR_BUFFER`?

Reply:
It was so in the original test. The trick is that the test might want to access some private fields - setting this flag makes all fields public.